api/controllers/case.js

```diff
@@ -34,35 +35,39 @@ var jsonStringify = require('json-pretty');
  *            ...
  *        }
  *     }
- *
+ * })
  */
 
 // TODO: figure out if the choropleth should show cases or all things
 
 router.get('/countsByCountry', function (req, res) {
-  let body = new Bodybuilder()
-  let bodyquery = body.aggregation('terms', 'geo_country', null, {size: 0}).size(0).build()
-  es.search({
-    index: 'pp',
-    type: 'case',
-    body: bodyquery
-  }).then(function (resp) {
-    var countryCounts = {}
-    let buckets = resp.aggregations.agg_terms_geo_country.buckets
-    for (let i in buckets) {
-      countryCounts[buckets[i].key] = buckets[i].doc_count
-    }
-    res.status(200).json({
-      OK: true,
-      data: {
-        countryCounts: countryCounts
-      }
+    db.query(
+        'select $1~.$3~, count($1~.$3~) from $1~, $2~ where $1~.$4~ = $2~.$5~ group by $1~.$3~;',
```

Comment:
Not now, but I'd hope there's a way of writing these queries in such a way that we use variable names that mean something to the reader, rather than have to do the mapping of indices to names, no?